tests/unit/test_cocip.py

```diff
@@ -1596,3 +1604,60 @@ def test_cocip_met_nonuniform(
     assert np.all(contrail_time > t0)
     assert np.any(contrail_time < t1)
     assert np.any(contrail_time > t1)
+
```

Comment:
Could you add two additional tests to check
- that `Cocip` runs with `improved_wake_vortex_ice_survival_fraction = True`? (No need to check values of outputs, just confirm that they're present.)
- that `CocipGrid` produces the expected error (https://docs.pytest.org/en/7.1.x/how-to/assert.html#assertions-about-expected-exceptions) when run with `improved_wake_vortex_ice_survival_fraction = True`?